### PR TITLE
Add base data pod angular components

### PR DESCRIPTION
**Pull request by justinvdm**
Same PR as #34, but against the correct branch.

### CODE REVIEW

**Comment by justinvdm** [comment]
@rudigiesler ready for review
@imsickofmaps ping so the PR is on your radar

File: static/coffee/services.coffee [hunk 1]
```diff
@@ -416,3 +416,15 @@ services.factory('UtilsService', ['$window', '$uibModal', ($window, $uibModal) -
...
+
+
+#=====================================================================
+# Pod API service
+#=====================================================================
+services.factory('PodApi', ['$window', '$http', ($window, $http) ->
+  new class PodApi
+    get: (podId, caseId = null) ->
+      caseId ?= $window.contextData.case_obj.id
+      $http.get("/pods/read/#{podId}/", {params: {case_id: caseId}})
```
Comment (justinvdm) [inline static/coffee/services.coffee]: From what I could tell from the bit of looking I did in cases/views, I'm guessing our pod api will have urls that look something like this. This is simple enough to change once we know how this api endpoint will look.

**Comment by miltontony** [comment]
👍

File: static/coffee/services.coffee (hunk 1 above)
Comment (miltontony) [inline static/coffee/services.coffee]: Cool

**Pull request merged**